### PR TITLE
master to master Sept 15

### PR DESCRIPTION
**Pull request by yairchn**

### CODE REVIEW

File: Forcing.pyx [hunk 1]
```diff
@@ -185,8 +197,20 @@ cdef class StochasticTropicalPlanet(ForcingBase):
...
+		if self.noise:
+			#print('Add stochastic forcing for vorticity equation')
+			F0=np.zeros(Gr.SphericalGrid.nlm,dtype = np.complex, order='c')
+			fr = spf.sphForcing(Pr.nlons,Pr.nlats,Pr.truncation_number,Pr.rsphere,
+				                Pr.Fo_noise_lmin, Pr.Fo_noise_lmax, Pr.Fo_noise_magnitude,
+				                correlation =Pr.Fo_noise_correlation, noise_type=Pr.Fo_noise_type)
+
+			forcing_noise = Gr.SphericalGrid.spectogrd(fr.forcingFn(F0))*Pr.Fo_noise_amplitude
+			sp_noise = Gr.SphericalGrid.grdtospec(forcing_noise.base)
+			PV.Vorticity.sp_forcing[:,Pr.n_layers-1] = sp_noise # lowest layer
```
Comment (yairchn) [inline Forcing.pyx]: leave a comment that it is only in the lowest layer

File: PrognosticVariables.pyx [hunk 2]
```diff
@@ -299,6 +297,7 @@ cdef class PrognosticVariables:
                 for i in range(nlm):
                     PV.Vorticity.tendency[i,k]  = (Vort_forc[i] - Divergent_momentum_flux[i]- w_vort_up[i] - w_vort_dn[i]
                                                      + Vort_sur_flux[i] + PV.Vorticity.ConvectiveFlux[i,k] + PV.Vorticity.sp_forcing[i,k])
+                    #PV.Vorticity.tendency[i,k]  =- Divergent_momentum_flux[i] + PV.Vorticity.sp_forcing[i,k] # barotropic vorticity eq. only
```
Comment (yairchn) [inline PrognosticVariables.pyx]: delete this comment

**Review by yairchn — COMMENTED** [review]
few small comments and we can merge

File: Forcing.pyx (hunk 1 above)
Comment (jschroettle) [reply Forcing.pyx]: added "forcing only in ..."

File: PrognosticVariables.pyx (hunk 2 above)
Comment (jschroettle) [reply PrognosticVariables.pyx]: done

**Pull request merged**